Examples/ResourceLeak/README.md

```diff
@@ -0,0 +1,28 @@
+## Procedure-local Bugs of Resource Leaks
+
+A simple example of **procedure-local bug of resource leak** is a resource leak in the context of incomplete code (e.g. without main) with inter-procedural reasoning. In this case, the usual notion that all resource allocated must be eventually freed does not apply in the context of incomplete code. **CSharpCodeAnalyzer** is designed to detect this kind of resource leaks by applying the following principle: **When a new object is allocated during the execution of a procedure, it is the procedure’s responsibility to either deallocate the object or make it available to its callers; there is no such obligation for objects received from the caller.** 
```

Comment:
I think this intro can just be reduced to something like, "The analyzer is able to detect resource leaks for the following scenarios:"

The reason is that the Readme should not provide "common" knowledge (our target audience isn't really novice users) -- there is Wikipedia for that. It should focus on the salient details exclusively, to focus the audience's attention.

Reply:
Resolved in update